packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -278,7 +278,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
     return (
       <ContextualMenu
         className={ css('ms-BaseButton-menuHost') }
-        isBeakVisible={ true }
+        isBeakVisible={ typeof menuProps.isBeakVisible === 'undefined' ? true : menuProps.isBeakVisible }
```

Comment:
it is a bit weird that we aren't spreading menuprops here... `{ ...menuProps }` and then layering the target on that. And css wrapping a string is useless... and menuProps.className is getting ignored, as is `data-automation-id`.